src/styles/blocks/calendar.scss

```
@@ -0,0 +1,54 @@
+.calendar {
+  position: absolute;
+  top: 50%;
+  display: flex;
+  flex-wrap: wrap;
+  width: $width-calendar;
+  gap: $gap-between-days;
+  transform: translateY(-50%);
```

```suggestion
  display: flex;
  flex-wrap: wrap;
  width: $width-calendar;
  gap: $gap-between-days;
```

Overthinked positioning. Just remove these props at all.